subsys/bluetooth/host/adv.c

```diff
@@ -661,6 +661,12 @@ static int hci_set_ad_ext(struct bt_le_ext_adv *adv, uint16_t hci_op,
 		return -EAGAIN;
 	}
 
+	if (total_len_bytes > bt_dev.le.max_adv_data_len) {
+		LOG_WRN("adv or scan rsp data too large (%d > max %d)", total_len_bytes,
```

Comment:
```suggestion
		LOG_ERR("adv or scan rsp data too large (%d > max %d)", total_len_bytes,
```

Reply:
This is a recoverable error, so based on the logging convention it's preferable to use a warning